storage/tests/unit/test__signing.py

```diff
@@ -69,7 +69,7 @@ def test_w_expiration_int(self):
         self.assertEqual(self._call_fut(123), 123)
 
     def test_w_expiration_long(self):
-        if six.PY3:
+        if not six.PY2:
             raise unittest.SkipTest("No long on Python 3")
```

Comment:
(nit)
With the changed condition, the error message can get outdated in Python 4, let's reword it

Reply:
How about "No long on Python 3+"?

09c70b6b8c632dd3413ea2824fc70872f18589ff

Reply:
Short and simple, I like it. :+1: